src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ConditionalAccess.cs

```diff
@@ -13,6 +13,11 @@ public override BoundNode VisitConditionalAccess(BoundConditionalAccess node)
             return RewriteConditionalAccess(node, used: true);
         }
 
+        public override BoundNode VisitLoweredConditionalAccess(BoundLoweredConditionalAccess node)
+        {
+            return node;
```

Comment:
I was tempted to put a `Debug.Assert(false);` here, but from discussion with Vlad, there may be some valid cases for lowering twice (for example synthesizing a foreach).